docker/influxd/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env ash
```

Comment:
What was wrong with bash?

Also, here the change is to `ash`, but in docker/flux/entrypoint.sh the change is to `sh`. We should at least be consistent.

Reply:
Bash isn't always available. It should be `sh`. I'll fix